pkg/database/vercode.go

```diff
@@ -75,36 +77,53 @@ func (v *VerificationCode) FormatSymptomDate() string {
 	return v.SymptomDate.Format("2006-01-02")
 }
 
+// IsCodeExpired checks to see if the actual code provides is the
+// short or long code and deteriminies if it is expired based on that.
+func (v *VerificationCode) IsCodeExpired(code string) bool {
+	now := time.Now()
```

Comment:
Should we be UTC-ing these?

Reply:
technically, I don't think it matters since time managed time zones, but it's good hygiene.